index.html

```diff
@@ -9,8 +9,63 @@
     <link rel="stylesheet" href="style/style.css" type="text/css">
 </head>
 <body>
-    <header>
-        
+    <header class="header">
+                <!-- navigation hamburger -->
+                <!-- <div class="mobile-container d-md-none">
+                    <div class="mobile-nav">
+                        <a href="#home"></a>
+                        <div class="menu-mobile">
+                                    <a class="nav-link col-md-4" href="#about-me">O mnie</a>
+                                    <a class="nav-link col-md-4" href="#projects">Projekty</a>
+                                    <a class="nav-link col-md-4" href="#contact">Kontakt</a>
+                        </div>
+                        <div class="mobile-menu_button icon">
+                            <img src="images/icons/icon-menu.png" alt="">
+                        </div>
+                    </div>
+                </div> -->
+
+                <nav class="d-md-none">
```

Comment:
to chyba jest kod skopiowany :). Czy jest on z Bootstrapa?

Reply:
tak skopiowany. kod wyzej ktory jest tutaj zakomentowany zostal usuniety w nastepnym commicie go nie bedzie

Reply:
A skąd go skopiowałeś :)? Czy jest to kod z Bootstrap4?